content/docs/replication/troubleshooting.md

```diff
@@ -17,3 +17,4 @@ description: >
 | PV and/or PVCs are not being created at the source/target cluster. If you check the controller's logs you can see `no such host` errors| Make sure cluster-1's API is pingable from cluster-2 and vice versa. If one of your clusters is OpenShift located in a private network and needs records in /etc/hosts, `exec` into controller pod and modify `/etc/hosts` manually. |
 | After upgrading to Replication v1.4.0, if `kubectl get rg` returns an error `Unable to list "replication.storage.dell.com/v1alpha1, Resource=dellcsireplicationgroups"`| This means `kubectl` still doesn't recognize the new version of CRD `dellcsireplicationgroups.replication.storage.dell.com` after upgrade. Running the command `kubectl get DellCSIReplicationGroup.v1.replication.storage.dell.com/<rg-id> -o yaml` will resolve the issue. |
 | To add or delete volumes in the existing SYNC Replication Group in PowerStore, you may encounter the error `The operation is restricted as sync replication session for resource <Replication Group Name> is not paused` | To resolve this, you need to pause the replication group, add the volume, and then resume the replication group (RG). The commands for the pause and resume operations are: `repctl --rg <rg-id> exec -a suspend`  `repctl --rg <rg-id> exec -a resume` |
+| To delete the last volume from the existing SYNC Replication Group in Powerstore , you may encounter the error 'failed to remove volume <Volume ID> from volume group: The operation cannot be completed on metro or replicated volume group <volume-group ID> because volume group will become empty after last members are removed' | To resolve this unmap volume group from protection policy which will delete the replication session . After that you can successfully  delete the last volume in that SYNC Replication group. | 
```

Comment:
There are some extra spaces in the line. Can you update the line as below:
To delete the last volume from the existing SYNC Replication Group in Powerstore, you may encounter the error 'failed to remove volume <Volume ID> from volume group: The operation cannot be completed on metro or replicated volume group <volume-group ID> because volume group will become empty after last members are removed' | To resolve this unmap the volume group from protection policy which will delete the replication session. After that, you can successfully delete the last volume in that SYNC Replication group. |

Reply:
Addressed the comment